### DISCUSSION TITLE
为什么很多杀毒软件把最新版alist视为病毒？

### DISCUSSION THREAD

**Discussion by MichaelChao02** [AlistGo/alist#3023] [General]
测试的文件为
https://github.com/alist-org/alist/releases/download/v3.8.0/alist-windows-386.zip

扫描结果为：
https://www.virustotal.com/gui/file/ce96b254284e68e247018e3168e5cdff7df74e5a2f2929527eeda5d9b6b0d028

69个杀毒软件中有12个认为是病毒

旧版本就没有被报毒

**Comment by anwen-anyi** [comment]
Windows用了upx压缩，会误报，具体的找不到了可以看开发者的回复 https://github.com/alist-org/alist/issues/3009#issuecomment-1379690254
